code/modules/mob/living/simple_animal/hostile/giant_spider/effects.dm

```diff
@@ -44,6 +45,23 @@
 	health -= damage
 	healthcheck()
 
+/obj/effect/spider/attack_hand(var/mob/living/carbon/human/user)
+	if(user.a_intent == I_HURT)
+		user.delayNextAttack(8)
+		user.visible_message("<span class='danger'>[user.name] claws at \the [src]!</span>", \
+			"<span class='danger'>You claw at \the [src]!</span>", \
+			"You hear something tear.")
+		health -= 2
```

Comment:
Some species have extra damage for melee attacks. It's in `punch_damage` on the species. Probably not very important but eh.